src/spf-explainer/templates/skeletons/record.vue

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright 2019 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+    <div>
+        <h3 :class="getClass()" :style="getSkeletonStyle(700, 800, 1.4)"></h3>
+        <hr>
+        <p :class="getClass()" :style="getSkeletonStyle(200, 400)"></p>
+        <hr>
+        <div class="record-group">
+            <table class="table skeleton-table">
+                <tbody>
+                    <tr v-for="_ in 5">
```

Comment:
Do we want to make this random like dns-tool?

Reply:
I personally think it looks better as static on the SPF tool, but I'm interested in your opinion on this.

Reply:
Keeping it static certainly makes designing the no-search state more predictable. Seems fine to keep it this way then 👍